### PR TITLE
chore: supabase edits

### PR DESCRIPTION
**Pull request by awalias**
updates language around Supabase's postgres compatibility (see: https://github.com/supabase/postgres), and adds Hipaa compliance a la https://supabase.com/security

### CODE REVIEW

File: content/blog/neon-vs-supabase.md [hunk 1]
```diff
@@ -104,8 +104,8 @@ CU (1vCPU, 4 GB RAM), and can scale from 0.25 to 8 CU.
 
 If you want a Postgres database without whistles and bells, Neon is almost the perfect database a developer would desire. It has serverless, branching, auto-scaling.
 
-If you don't care too much about the Postgres genre and want to build a full-stack application, Supabase has everything
-you need. It has database, auth, API. In the future, it could also be possible that Supabase offers Neon as a database option.
+If you want 100% Postgres compatibility or are looking to build a full-stack application, Supabase has everything
```
Comment (tianzhou) [inline content/blog/neon-vs-supabase.md:107]: https://supabase.com/docs/guides/database/postgres/roles-superuser

99% or even 99.9% is fine, however, saying it 100% misses the mark as compared with self-hosted PG.

**Review by tianzhou — CHANGES_REQUESTED** [review]
One comment. Otherwise LGTM.

File: content/blog/neon-vs-supabase.md (hunk 1 above)
Comment (awalias) [reply content/blog/neon-vs-supabase.md:107]: ah that's an interesting point, it's probably a matter of definition but no worries I will reformulate 👍

Comment (awalias) [reply content/blog/neon-vs-supabase.md:107]:
```suggestion
If you're looking for a dedicated Postgres instance or are looking to build a full-stack application, Supabase has everything
```

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**